### PR TITLE
Feat/nutanix

### PR DESCRIPTION
**Pull request by radireddy**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | yes
| Tickets          | Fix #MANAGER-6011
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Comment by radireddy** [comment]
Initial PR https://github.com/ovh/manager/pull/6306
Some of the review comments as been addressed as part of this PR

**Comment by radireddy** [comment]
> SonarCloud Quality Gate failed.    Quality Gate failed
> 
> [Bug](https://sonarcloud.io/project/issues?id=ovh_manager&pullRequest=6314&resolved=false&types=BUG) [C](https://sonarcloud.io/project/issues?id=ovh_manager&pullRequest=6314&resolved=false&types=BUG) [2 Bugs](https://sonarcloud.io/project/issues?id=ovh_manager&pullRequest=6314&resolved=false&types=BUG) [Vulnerability](https://sonarcloud.io/project/issues?id=ovh_manager&pullRequest=6314&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=ovh_manager&pullRequest=6314&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=ovh_manager&pullRequest=6314&resolved=false&types=VULNERABILITY) [Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=ovh_manager&pullRequest=6314&resolved=false&types=SECURITY_HOTSPOT) [E](https://sonarcloud.io/project/security_hotspots?id=ovh_manager&pullRequest=6314&resolved=false&types=SECURITY_HOTSPOT) [1 Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=ovh_manager&pullRequest=6314&resolved=false&types=SECURITY_HOTSPOT) [Code Smell](https://sonarcloud.io/project/issues?id=ovh_manager&pullRequest=6314&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=ovh_manager&pullRequest=6314&resolved=false&types=CODE_SMELL) [5 Code Smells](https://sonarcloud.io/project/issues?id=ovh_manager&pullRequest=6314&resolved=false&types=CODE_SMELL)
> 
> [No Coverage information](https://sonarcloud.io/component_measures?id=ovh_manager&pullRequest=6314) No Coverage information [](https://sonarcloud.io/component_measures?id=ovh_manager&pullRequest=6314&metric=new_duplicated_lines_density&view=list) [0.9% Duplication](https://sonarcloud.io/component_measures?id=ovh_manager&pullRequest=6314&metric=new_duplicated_lines_density&view=list)

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/constants.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export const NO_OS_INSTALLED_REGEX = /^none_\d{2}?$/;
+export const TRAVAUX_LINK = 'http://travaux.ovh.com/';
```
Comment (marie-j) [inline packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/constants.js:2]: Url has changed

**Review by marie-j — CHANGES_REQUESTED** [review]

**Comment by marie-j** [comment]
Could you please check Sonar report ? There might be a few issues we can fix

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/constants.js (hunk 1 above)
Comment (radireddy) [reply packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/constants.js:2]: https://www.status-ovhcloud.com/
This is the new URL?

**Review by anooparveti — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/constants.js (hunk 1 above)
Comment (antleblanc) [reply packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/constants.js]: Yes it is!

Comment (antleblanc) [reply packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/constants.js]: Yes it is!

**Review by ghyenne — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by marie-j — APPROVED** [review]

**Pull request merged**